core/src/dynamic.rs

```diff
@@ -66,8 +63,8 @@ impl DecodedValueThunk {
     pub fn encoded(&self) -> &[u8] {
         &self.scale_bytes
     }
-    /// Decode the SCALE encoded storage entry into a dynamic [`DecodedValue`] type.
-    pub fn to_value(&self) -> Result<DecodedValue, Error> {
+    /// Decode the SCALE encoded storage Value into a dynamic [`DecodedValue`] type.
```

Comment:
I think "storage entry" makes more sense than "storage Value" (the Value is what it would be decoded into possibly, but not here)


```suggestion
    /// Decode the SCALE encoded storage entry into a dynamic [`DecodedValue`] type.
```